contents/docs/migrate/migrate-to-cloud.mdx

```diff
@@ -108,7 +103,17 @@ python3 ./migrate.py \
 > - This script adds a `$lib` property of `posthog-python`, overriding any `$lib` property already set.
 > - If the script fails for some reason, just run it again with the latest cursor. There are some transient issues that can happen that are solved by re-running the script.
 
-## Switching tracking in you product
+### Between Cloud instances (e.g. US Cloud to EU CLoud)
+
+You will need to ask us to do the events migration for you by [raising a support ticket in the app](https://app.posthog.com/home#supportModal). Please note that this option is only available to paying customers as it requires significant engineering time. 
```

Comment:
Do we want to explicitly call out the teams plan here?

Reply:
I was thinking to start with all paying customers and then we can narrow the scope further from there if it's still annoying to manage? I think restricting to Teams plan only might be annoying for customers say spending $1k/mo but aren't on the Teams plan. 

Or we could generically say 'we'll consider this on a case by case basis' and then @simfish85 use his discretion? Might be the easiest option for now.

Reply:
I'd say docs say teams plan only and then customers likely will talk to CS anyway to ask for an exception?

Reply:
I think it's fairly pointless asking for Teams plan as they could pay once for it and then cancel which doesn't really help our revenue numbers.  I'd say draw the line at people who've actually paid anything for now and see if that helps lower the volume.

Reply:
Let's merge this for now, and then revisit in a couple of weeks if we need to tweak further, as it's still significantly better than what we have.